2.js

```diff
@@ -0,0 +1,86 @@
+const readline = require('readline')
+
+const rl = readline.createInterface({
```

Comment:
I recommend the use of intention revealing identifiers (`rl` might not be clear when used later in the code).